lms/services/lti_grading/_v11.py

```diff
@@ -44,11 +44,6 @@ def record_result(self, grading_id, score=None, pre_record_hook=None):
         if pre_record_hook:
             request = pre_record_hook(score=score, request_body=request)
 
-            if not isinstance(request, dict):
```

Comment:
I don't see a reason to type check this and not any other function in our codebase.